File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        int l1 = s.length();
        int l2 = t.length();
        if(l1!=l2)
            return false;
        sort(s.begin(),s.end());
        sort(t.begin(),t.end());
        for(int i=0 ; i<l1 ; i++)
            if(s[i]!=t[i])
                return false;
        return true;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length()!=t.length())
            return false;
        int[] arr = new int[27];
        for(int i=0 ; i<s.length()  ; i++){
            arr[s.charAt(i) - 'a']++;
            arr[t.charAt(i) - 'a']--;
        }
        for(int i=0 ; i<27 ; i++){
            if(arr[i]!=0)
                return false;
        }
        return true;

    }
}